pkg/api_builder/lib/api_builder.dart

```diff
@@ -76,15 +92,10 @@ abstract class $utilities {
       }
       throw FormatException('payload must always be a JSON object');
     } on FormatException {
-      throw ResponseException(
-        400,
-        json.fuse(utf8).encode({
-          'message': 'malformed JSON payload',
-        }),
-        {
-          'content-type': 'application/json; charset="utf-8"',
-          'x-content-type-options': 'nosniff',
-        },
+      throw ApiResponseException(
+        status: 400,
+        code: 'InvalidInput',
```

Comment:
invalid-input? not sure if it is important... maybe not at all.
Or `InvalidInput` could be emitted on `InvalidInputException`?

Reply:
I don't care much what the codes are, as long as we try to be consistent :) hehe

There is only a very few of these codes that we'll ever what to handle.

Note: I do prefer `CamelCase` for these error codes as they will be identifiers in clients. We could make an enum of all the error codes, but I think we'll be fine if we just document the ones clients need to handled in the API docs.

Oh, yes, let's say that the comment infront of the `@EndPoint...` annotation is the API documentation.. later we can code-gen some API docs from those :) hehe